### PR TITLE
Changed title, added image  captions to gallery

### PR DESCRIPTION
**Pull request by lattai**

### CODE REVIEW

**Comment by lattai** [comment]
I accidentally forgot to merge this pr before making a new commit! I'd really appreciate it if the changes in the commit were reviewed. Thanks so much all!

The link is [leahattai-step-2020.appspot.com](url)

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -50,7 +43,7 @@ var funFacts = " - I got into CS by doing robotics in middle school!"+
 
 // Opens information for whichever tab was clicked. 
 function openTab(info) {
-    const SUBPARAGRAPH_CONTAINER = document.getElementById('about-container');
+	const SUBPARAGRAPH_CONTAINER = document.getElementById('about-container');
```
[the remark this answers is not in the document]
Comment (lattai) [reply portfolio/src/main/webapp/script.js:46]: Will fix the spacing! As for the link, try this: https://leahattai-step-2020.appspot.com/

Comment (lzoebaker) [inline portfolio/src/main/webapp/script.js:46]: indent looks a bit off here.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -60,7 +63,7 @@ h1 {
     font-family: 'Permanent Marker';
     text-shadow: 1px 2px var(--red);
     color: var(--olive);
-    font-size: 30px;
+    font-size: vaar(--heading1);
```
Comment (lzoebaker) [inline portfolio/src/main/webapp/style.css:66]: typo, vaar -> var.

**Pull request merged**